apps/client/src/components/globalSettings/UnivariateCellPlot.vue

```diff
@@ -0,0 +1,288 @@
+<template>
```

Comment:
Same comment with order.
1. script
2. template
3. style

This is just a consistency thing for the project. I like script at top because it's easy to jump to where I add imports when coding.